deepchecks/base/display_suite.py

```diff
@@ -152,10 +156,30 @@ def _display_suite_widgets(unique_id: str,
         others_tab.children = [widgets.HTML(_NO_OUTPUT_TEXT)]
     condition_tab.children = condition_tab_children
     checks_wo_tab.children = checks_wo_tab_children
-    display(tab)
 
+    tab_css = '<style>.jupyter-widgets.widget-tab > .p-TabBar .p-TabBar-tab {flex: 0 1 auto}</style>'
+    page = widgets.VBox()
+    page.children = [widgets.HTML(summary), widgets.HTML(tab_css), tab]
+    if as_html:
+        curr_path = os.path.dirname(os.path.abspath(__file__))
+        basename, ext = ('output', 'html')
+        actualname = f'{basename}.'
+        c = itertools.count()
+        next(c)
+        while os.path.exists(actualname):
+            actualname = f'{basename} ({str(next(c))}).{ext}'
```

Comment:
```suggestion
        curr_path = os.path.dirname(os.path.abspath(__file__))
        
        actualname = html_file_name #passed from user
        
        if not actualname:
            default_basename, ext = ('output', 'html')
            actualname = f'{default_basename}.'
            c = itertools.count()
            next(c)
            while os.path.exists(actualname):
                actualname = f'{basename} ({str(next(c))}).{ext}'
```

to allow user to pass filename

Reply:
I agree. I think it shouldn't be a name but a path (like pandas.to_html), but it should be optional and behave exactly as @JKL98ISR wrote it if the user didn't specify a path.

Reply:
done